packages/xo-web/src/common/xo/index.js

```diff
@@ -615,6 +615,34 @@ export const addHostsToPool = pool =>
     })
   })
 
+export const enableAdvancedLiveTelemetry = async host => {
+  const isConfiguredToReceiveStreaming = await _call(
+    'netdata.isConfiguredToReceiveStreaming',
+    { host: host.id }
+  )
```

Comment:
I guess you could inline it. If you purposely extracted it for readability/documentation, then I'm fine with it.

Reply:
Yes I extracted it for readability :+1: